docs/iris/src/whatsnew/contributions_3.0.0/incompatiblechange_2019-Nov-13_move_experimental_equalise_cubes.txt

```diff
@@ -0,0 +1,2 @@
+* The :func:`iris.experimental.equalise_cubes.equalise_attributes` function has been migrated into
+  the core of Iris. Please use the :func:`iris.util.equalise_attributes` function instead.
```

Comment:
I'm not sure "core of Iris" means that much to someone outside the dev team.
Maybe it is clearer to say:
* The :func:`iris.experimental.equalise_cubes.equalise_attributes` function has been moved from the  :mod:`iris.experimental` module into :mod:`iris.util` module. Please use the :func:`iris.util.equalise_attributes` function instead.